YoutubeExplode/Playlists/Playlist.cs

```diff
@@ -37,13 +39,15 @@ public class Playlist : IPlaylist
         string title,
         Author? author,
         string description,
-        IReadOnlyList<Thumbnail> thumbnails)
+        IReadOnlyList<Thumbnail> thumbnails,
+        string? url = null)
```

Comment:
We'd have to pass `url` in a separate overload (with and without `url`) due to backwards compatibility

Reply:
do you mean removing it as an optional parameter and making another constructor overload?

Reply:
> do you mean removing it as an optional parameter and making another constructor overload?

Yeah. That's because other libraries that depend on YoutubeExplode will fail to bind to this constructor since the signature has changed.